src/connection.cpp

```diff
@@ -1284,11 +1309,12 @@ namespace oxen::quic
             std::shared_ptr<IOContext> ctx,
             const std::vector<std::string>& alpns,
             std::chrono::nanoseconds handshake_timeout,
+            std::optional<ustring> remote_pk,
             ngtcp2_pkt_hd* hdr)
     {
         log::trace(log_cat, "{} called", __PRETTY_FUNCTION__);
         std::shared_ptr<Connection> conn{
-                new Connection{ep, scid, dcid, path, std::move(ctx), alpns, handshake_timeout, hdr}};
+                new Connection{ep, scid, dcid, path, std::move(ctx), alpns, handshake_timeout, remote_pk, hdr}};
```

Comment:
Can usefully `std::move` the `remote_pk` argument here (I think everything else here not already being moved is trivially copyable).